### PR TITLE
Added tip to change "My Organization" to "ArcGIS Online"

### PR DESCRIPTION
**Pull request by garysheppardjr**

### CODE REVIEW

File: data/view_in_a_map/lab.md [hunk 1]
```diff
@@ -21,4 +21,4 @@ In this lab you will add your own layers and some existing publicly available la
 6. Save your map. 
 
 ###Bonus
-* In the Map Viewer select `Add` > `Search for Layers` > `In: ArcGIS Online` to load other pre-existing layers into your map. HINT: Try searching for weather, rain or wind.
\ No newline at end of file
+* In the Map Viewer select `Add` > `Search for Layers` > `In: ArcGIS Online` to load other pre-existing layers into your map. HINT: Try searching for weather, rain or wind. You might try changing `In: My Organization` to `In: ArcGIS Online` to see more layers.
```
Comment (jgravois) [inline data/view_in_a_map/lab.md]: we need to be a little clearer here.  if they start `In: ArcGIS Online`, then the appropriate change is *to* `In: My Organization` (which filters out outside results)

Comment (garys-esri) [inline data/view_in_a_map/lab.md]: @jgravois, I get zero results when I search for weather, rain, or wind in the efsbs org. That's why I assumed that one could find these layers outside the organization. Are there some layers that are not shared with the whole org?

Comment (jgravois) [inline data/view_in_a_map/lab.md]: searching across ArcGIS Online will definitely return more results than a search within an individual Organization.  

i was only pointing out that your current wording seems unclear.  the original step asks them to select `In: ArcGIS Online` and then you mention switching from `In: My Organization` to `In: ArcGIS Online` (where they already should have been)

Comment (garys-esri) [inline data/view_in_a_map/lab.md]: D'oh. You're right. Sorry. I'll close the pull request.

**Pull request closed without merging**